### DISCUSSION TITLE
EASIEST way to shutdown PC remotely!

### DISCUSSION THREAD

**Discussion by ghost** [AlexGustafsson/homebridge-wol#145] [General]
Hey everyone! 

I just wanted to share a very easy way of shutting down your PC.
This works on all platforms (Windows, Linux, MacOS)

It's called Magic Packet Server.
https://apps.somenkov.ru/magic-packet/server/

With this program installed on your PC, you can simply shut it down by sending a POST request to your PC.

So if you have it installed, you can shut down with:

![afbeelding](https://user-images.githubusercontent.com/39121384/108592777-d3635a80-736f-11eb-9d4a-06b3659f3e77.png)

With this you don't have to mess with ssh, user accounts, passwords and all that complicated stuff.
Hope you find it helpful :)

**Comment by AtomikHeh** [comment]
Still works as of today, this is the best alternative that you can possibly find. It's so straightforward and easy to use compared to what you're supposed to do without that piece of software right there.
